env-windows.yml

```diff
@@ -0,0 +1,28 @@
+name: vpn
+channels:
+  - conda-forge
+dependencies:
+  - clang=16.0.6
+  - clang-tools=16.0.6
+  - clangxx=16.0.6
+  - llvm=16.0.6
+  - llvmdev=16.0.6
+  - lld=16.0.6
+  - python=3.9
+  - nodejs=18.16.*
+  - pip=22.3.1
+  - rust=1.75
+  - rust-std-armv7-linux-androideabi=1.75
+  - rust-std-x86_64-linux-android=1.75
```

Comment:
We don't support compiling android on windows, so no need to have those :)

Reply:
Huzzah!